project/plugins.sbt

```diff
@@ -1,5 +1,3 @@
-addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.3")
```

Comment:
https://get-coursier.io/docs/sbt-coursier

> Note that if you are using sbt 1.3.x or later, you shouldn't need to add sbt-coursier to your build, except in very specific cases. sbt 1.3.x ships with coursier support out-of-the-box. That support is enabled by default.

I should have really removed this in https://github.com/guardian/elastic-search-monitor/pull/9 - I only noticed the duplication here because I couldn't download Scala 3 until I removed this plugin!